deployments/cognito-rds-s3/terraform/main.tf

```diff
@@ -249,6 +280,7 @@ module "kubeflow_components" {
 
   tags = local.tags
 
+
```

Comment:
is this coming from terraform fmt?

if you have not run it already, can you run it? thx

Reply:
ran it